src/core/server/saved_objects/import/extract_errors.test.ts

```diff
@@ -21,13 +21,32 @@ import { SavedObject } from '../types';
 import { extractErrors } from './extract_errors';
 
 describe('extractErrors()', () => {
-  test('returns empty array when no errors exist', () => {
+  const savedObjectsClient = {
+    errors: {} as any,
+    bulkCreate: jest.fn(),
+    bulkGet: jest.fn(),
+    create: jest.fn(),
+    delete: jest.fn(),
+    find: jest.fn(),
+    get: jest.fn(),
+    update: jest.fn(),
+  };
+
+  test('returns empty array when no errors exist', async () => {
     const savedObjects: SavedObject[] = [];
-    const result = extractErrors(savedObjects, savedObjects);
+    const result = await extractErrors(savedObjects, savedObjects, savedObjectsClient);
     expect(result).toMatchInlineSnapshot(`Array []`);
   });
 
-  test('extracts errors from saved objects', () => {
+  test('extracts errors from saved objects', async () => {
+    const clientResponse = {
+      saved_objects: [
+        { id: '2', type: 'dashboard', attributes: { title: 'My Dashboard 2' } },
```

Comment:
Can you change these titles to something like 'Existing Dashboard 2' otherwise we're not testing the existing title finding logic you added. I think this will also surface the the bug where we resolve the existing title of "My Dashboard 3" even though this wasn't a conflict error.